components/post_view/reaction/reaction.jsx

```diff
@@ -245,13 +307,23 @@ export default class Reaction extends React.PureComponent {
                 >
                     <span className='d-flex align-items-center'>
                         <span
-                            className='post-reaction__emoji emoticon'
+                            className='Reaction__emoji emoticon'
```

Comment:
can we use the previous class name format here too? I am 3/5 to not use capital letters :)

Reply:
I was following the widget component pattern, but I’m happy to standardize going forward. If we change these then we should probably change the widgets : )

Reply:
Leaving as capitalized for now because I'd like to have a discussion about this as a team.